b_server.md

```diff
@@ -15,245 +19,372 @@ If the server is responding returns:
 
-### Checking whether an org admin username is already in use
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/complete_signup
+```
 
-_GET https://sandbox.b.jlinclabs.net/api/orgs/exists/{name-to-check}_
+Creates an organization, organization admin, and stores a session.
+
+_Post body:_
+```json
+{
+  "name": "Bob Co",
+  "email": "neworg@example.com",
+  "password": "passwordbird",
+  "passwordConfirmation": "passwordbird",
+  "signupToken": "eV7i1ldl8HoeVFBDtL9paUbS2kFl3rlB",
+  "salesforce": true, // whether or not the organization has a Salesforce connection
+  "keepMeLoggedIn": true
+}
+```
 
-If the name (after lowercasing) is available (i.e. does not exist), the reply result will be `false`. If the name does exist the reply will be `true`.
+Returns a session ID and initial organization details.
 
+_Response:_
 ```json
 {
  "success": true,
- "result": false
+ "sessionId": "5b2dbdc40a22ecc8ca2e12c3da0f0319",
+ "organization": {
+    "apikey": "bobco",
+    "logo": null,
+    "icon": null,
+    "name": "Bob Co",
+    "dpo_email": "neworg@example.com",
+    "domain": null,
+    "contact_phone": null,
+    "contact_phone_2": null,
+    "address": null,
+    "city": null,
+    "post_code": null,
+    "country": null,
+    "communication_channels": {},
+    "requested_data": {},
+    "consents": {},
+    "salesforce": false,
+    "description": null,
+    "state": null,
+    "public": false,
+    "banner": null,
+    "upsell_description": null,
+    "public_allowed": false,
+    "published": false,
+    "marketplace_tags": [],
+    "industries": [],
+    "brands": [],
+    "public_key": "Vi6djLwwoiga_XWE93sdbYPbgP2jxjCjHDNfq5qFcRg",
+    "dcust_id": "Vi6djLwwoiga_XWE93sdbYPbgP2jxjCjHDNfq5qFcRg",
+    "api_secret": "457b855bbfb36a3bd5df045066f34de40b2db65314cef8b9d83ae272f40fc127",
+    "vendor_name": "bobco"
+  }
 }
 ```
-{% include starsep.html %}
 
-### Adding an org admin to an existing org
+{% include starsep.html %}
 
-_POST https://sandbox.b.jlinclabs.net/api/orgs/register_
+<a name="login"></a>
+### Logging in as an organization admin
 
-Requires a signup token created out-of-band
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/login
+```
 
+_Post body:_
 ```json
 {
-  "email": "newadmin@example.com",
-  "password": "foobar",
-  "password_confirmation": "foobar",
-  "signuptoken": "5e1668d7fffaadabf0474088e..."
+ "email": "neworg@example.com",
+ "password": "passwordbird",
+ "keepMeLoggedIn": true
 }
 ```
 
-Success returns the email and a session ID:
+A successful login returns a session ID:
 
+_Response:_
 ```json
 {
  "success": true,
- "email": "newadmin@example.com",
  "sessionID": "5e1668d7fffaadabf0474088e..."
 }
 ```
 
+The sessionId provided in the response should be used as the value of the
+`Session-Id` header of [authenticated requests](#authenticated-requests) section that appears later in this document.
+
 {% include starsep.html %}
 
-### Logging in an org admin
+<a name="verify-invite-token"></a>
```

Comment:
these anchor names should all be __exactly__ the same as the header, lowercase with dashes. Please fix all of them.

I would also change all the headers to something more like a terse actions name as apposed to some long english description. So `# Login` instead of `#  Logging in an org admin`

Reply:
this is also just a general programming best practice. Never have two names for the same thing when one works.